.github/pull_request_template.md

```diff
@@ -6,6 +6,19 @@
 
 (Details)
 
+## Does this PR affect the documentation?
+
+- [ ] No. You can skip the rest of this section.
+- [ ] Yes. Make sure the affected pages / sections render correctly by following the steps below.
+
+Steps:
+
+1. Make sure the status of the `ci/circleci: build_doc` check is successful. If no, fix it.
+2. Click `Details` to open the job page of CircleCI.
+3. Click the `Artifacts` tab.
+4. Click `docs/build/html/index.html`.
+5. Find the affect pages / sections and make sure they render correctly.
```

Comment:
This section should look like this:

![image](https://user-images.githubusercontent.com/17039389/140912410-1dfac7a2-e53b-442c-b786-1fba79994caa.png)